.github/release-drafter.yml

```diff
@@ -1 +1,2 @@
-_extends: .github
\ No newline at end of file
+_extends: .github
+tag-template: Apimap-$NEXT_MINOR_VERSION
```

Comment:
I guess you are you attempting to disable CD and use manual (local) `maven-release-plugin`?